Swivl/Sniffs/Commenting/DoctrineEntitySniff.php

```diff
@@ -13,6 +13,11 @@
  */
 class DoctrineEntitySniff extends AbstractVariableSniff
 {
+    private const CLASS_SUFFIX = '::class';
+
+    // 7 simbols contain in '::class'
+    private const CLASS_SUFFIX_START_POSSITION = -7;
```

Comment:
I think it's better to use CLASS_SUFFIX_LENGTH as it will be more clear here without reading further usage of this constant and therefore there is no need in the comment:
```suggestion
    private const CLASS_SUFFIX_LENGTH = 7;
```